local-cli/server/server.js

```diff
@@ -12,6 +12,7 @@ const chalk = require('chalk');
 const formatBanner = require('./formatBanner');
 const path = require('path');
 const runServer = require('./runServer');
+const findSymlinksPaths = require('./findSymlinksPaths');
```

Comment:
I'd say I want to return an array of resolved symlink paths from this function and apply them wherever it'll be needed.